holmes/plugins/toolsets/kafka.py

```diff
@@ -538,6 +538,26 @@ def prerequisites_callable(self, config: Dict[str, Any]) -> Tuple[bool, str]:
             logging.exception("Failed to set up Kafka toolset")
             return False, str(e)
 
+    def init_config(self):
+        kafka_config = KafkaConfig(**self.config)
+
+        for cluster in kafka_config.kafka_clusters:
+            admin_config = {
+                "bootstrap.servers": cluster.kafka_broker,
+                "client.id": cluster.kafka_client_id,
+            }
+
+            if cluster.kafka_security_protocol:
+                admin_config["security.protocol"] = cluster.kafka_security_protocol
+            if cluster.kafka_sasl_mechanism:
+                admin_config["sasl.mechanisms"] = cluster.kafka_sasl_mechanism
+            if cluster.kafka_username and cluster.kafka_password:
+                admin_config["sasl.username"] = cluster.kafka_username
+                admin_config["sasl.password"] = cluster.kafka_password
+
+            client = AdminClient(admin_config)
+            self.clients[cluster.name] = client  # Store in dictionary
```

Comment:
The code here is a copy paste of the code in `prerequisites_callable`. It should be extracted in its a private method to avoid duplication.